oic.baseResource/oic.baseResource.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "http://openinterconnect.org/iotdatamodels/schemas/oic.baseResource.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Copyright (c) 2016 Open Connectivity Foundation, Inc. All rights reserved.",
+    "title": "Base Resource",
+    "definitions": {
+        "oic.r.baseresource": {
```

Comment:
Thanks! I'll add the RAML in a subsequent PR. Added this issue to track adding that later: https://github.com/openT2T/translators/issues/54 so for now this is #WontFix in this PR (hope I got that terminology right?)